packages/react-accordion/src/utils/descendants.tsx

```diff
@@ -0,0 +1,399 @@
+import * as React from 'react';
```

Comment:
Could you please add a comment explaining where this file comes from and what the future plan is?